diff --git a/atom/renderer/atom_renderer_client.cc b/atom/renderer/atom_renderer_client.cc
index 2794ce0ad..6beabbab5 100644
--- a/atom/renderer/atom_renderer_client.cc
+++ b/atom/renderer/atom_renderer_client.cc
@@ -81,8 +81,8 @@ void AtomRendererClient::DidCreateScriptContext(
     content::RenderFrame* render_frame) {
   // Only allow node integration for the main frame, unless it is a devtools
   // extension page.
-  if (!render_frame->IsMainFrame() && !IsDevToolsExtension(render_frame))
-    return;
+  //if (!render_frame->IsMainFrame() && !IsDevToolsExtension(render_frame))
+  //  return;
 
   injected_frames_.insert(render_frame);
 
diff --git a/atom/renderer/atom_sandboxed_renderer_client.cc b/atom/renderer/atom_sandboxed_renderer_client.cc
index ca66298fc..a8df133b2 100644
--- a/atom/renderer/atom_sandboxed_renderer_client.cc
+++ b/atom/renderer/atom_sandboxed_renderer_client.cc
@@ -149,8 +149,8 @@ void AtomSandboxedRendererClient::DidCreateScriptContext(
     content::RenderFrame* render_frame) {
   // Only allow preload for the main frame or
   // For devtools we still want to run the preload_bundle script
-  if (!render_frame->IsMainFrame() && !IsDevTools(render_frame))
-    return;
+  //if (!render_frame->IsMainFrame() && !IsDevTools(render_frame))
+  //  return;
 
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
   base::FilePath preload_script_path =
@@ -184,8 +184,8 @@ void AtomSandboxedRendererClient::WillReleaseScriptContext(
     v8::Handle<v8::Context> context,
     content::RenderFrame* render_frame) {
   // Only allow preload for the main frame
-  if (!render_frame->IsMainFrame())
-    return;
+  // if (!render_frame->IsMainFrame())
+  //  return;
 
   auto* isolate = context->GetIsolate();
   v8::HandleScope handle_scope(isolate);
